threejs/src/index.js

```diff
@@ -1,26 +1,26 @@
 import * as THREE from 'three';
 import { fetchNFT, fetchActiveAd, sendMetric } from '../../utils/networking'
+import formats from '../../utils/formats';
 
 export default class ZestyAd extends THREE.Mesh {
   /**
    * @constructor
    * @param {string} adSpace The adSpace ID
    * @param {string} creator The wallet ID of the creator
-   * @param {string} defaultAd The format of the default ad, defaults to square
-   * @param {Number} width Width of the ad, defaults to 2
+   * @param {string} adFormat The format of the default ad, defaults to square
    * @param {Number} height Height of the ad, defaults to 3
    * @param {THREE.WebGLRenderer} renderer Optional field to pass in the WebGLRenderer in a WebXR project
    */
-  constructor(adSpace, creator, defaultAd = 'square', width = 2, height = 3, renderer = null) {
+  constructor(adSpace, creator, adFormat = 'square', height = 1, renderer = null) {
```

Comment:
Instead of using the literal `square` maybe import a const `defaultFormat` from `formats.js`. What do you think?